### ISSUE TITLE
Azure API didn't work

### ISSUE THREAD

**Issue by kotobuki09** [josStorer/chatGPTBox#645]
I got error while try to use Azure API. My deployment is exist for long time. It's shouldn't be an issue here. Do anyone know why is that?

Error:

{
"error": {
"code": "DeploymentNotFound",
"message": "The API deployment for this resource does not exist. If you created the deployment within the last 5 minutes, please wait a moment and try again."
}
}

**Comment by josStorer** [comment]
Was it normal before, or is this your first time trying to use Azure API in ChatGPTBox? I think there may be some setting issue with the Deployment you created.

**Comment by kotobuki09** [comment]
Yes, I used to work for sure. However, I use the same resources this time. And it's show this error like above

**Comment by josStorer** [comment]
could you give me the screenshots of your azure deployment name and chatgptbox settings

**Comment by PeterDaveHello** [comment]
It looks like you need to check your model deployment on Azure OpenAI, I just test this function and it works perfectly.

**Comment by milindscloud** [comment]
Hello kotobuki09 bro , i am really interested in your IMS for VLC project. Is it possible for us to chat ,if yes please message at milindmini2002@gmail.com

**Comment by PeterDaveHello** [comment]
Replied and closed by gpt-5.6-luna xhigh:

> Azure OpenAI remains supported. The reported DeploymentNotFound response requires the Azure deployment name and settings, which were not provided for a reproducible extension defect.
